pyop2/caching.py

```diff
@@ -234,66 +229,3 @@ def _cache_key(cls, *args, **kwargs):
     def cache_key(self):
         """Cache key."""
         return self._key
-
-
-class DiskCached(Cached):
```

Comment:
Ah, I see this is merged into `TSFCKernel`, probably the only user of `DiskCached`.